### PR TITLE
Bug fix recipie deletions  issue #43

### PR DESCRIPTION
**Pull request by sadath2001**
hi @AlfiyaSiddique ,

I have closed the previous PR #54 since it was very much outdated branch.

Here is the fix as below:

https://github.com/user-attachments/assets/48b97335-b812-405f-85a1-143fe6beb855

Kindly let me know if any change needed.

If not kindly merge this under GSSOC ext and hacktoberfest.

### CODE REVIEW

File: frontend/src/Pages/Dashboard.jsx [hunk 1]
```diff
@@ -29,17 +32,27 @@ const Dashboard = () => {
                 }
             )
             .then((res) => {
-                setRecipe(res.data.recipes);
+                console.log("Recipes fetched:", res.data.recipes);
```
Comment (AlfiyaSiddique) [inline frontend/src/Pages/Dashboard.jsx:35]: remove this console

**Comment by AlfiyaSiddique** [comment]
@sadath2001 one improvement and a comment please fix it and rest looks good tome!
Instead of using browser prompt can you make a custom prompt or using any package which matches with the site design

**Review by sadath2001 — COMMENTED** [review]
removed the console.log

**Comment by sadath2001** [comment]
hi @AlfiyaSiddique , i have made the changes

**Comment by sadath2001** [comment]
> @sadath2001 one improvement and a comment please fix it and rest looks good tome! Instead of using browser prompt can you make a custom prompt or using any package which matches with the site design

i'm currently learning react, so this was the best i could do as of now. i'll try to do better next time

**Comment by AlfiyaSiddique** [comment]
Okay @sadath2001

**Review by AlfiyaSiddique — APPROVED** [review]

**Comment by AlfiyaSiddique** [comment]
issue #43

**Pull request merged**